yt/yt/library/containers/cri/cri_executor.cpp

```diff
@@ -96,6 +96,15 @@ class TCriProcess
 
     void Kill(int /*signal*/) override
     {
+        if (Finished_) {
```

Comment:
I don't see this field in this class.

Are you sure, that this patch is working?

Reply:
That protected field in parrent class TProcessBase

https://github.com/ytsaurus/ytsaurus/blob/2744cdef14773f33eb7db1de465c56a99bd1eae6/yt/yt/library/process/process.h#L57